pyomo/contrib/parmest/utils/model_utils.py

```diff
@@ -124,14 +126,18 @@ def convert_params_to_vars(model, param_names=None, fix_vars=False):
     if len(substitution_map) == 0:
         return model
 
-    # Update the list of param_names if the parameters were indexed
-    if len(indexed_param_names) > 0:
-        param_names = indexed_param_names
+    # Update the list of param_CUIDs if the parameters were indexed
+    if len(indexed_param_CUIDs) > 0:
+        param_CUIDs = indexed_param_CUIDs
```

Comment:
Why do we replace `param_CUIDs` here instead of extending it?

Reply:
I think we have to replace it because it might not have the indexed variables, only the main variable.  For example "theta" (param_CUIDs) vs. "theta[asymptote]" and "theta[rate_constant]" (indexed_param_CUIDs).